### DISCUSSION TITLE
helm细节指导

### DISCUSSION THREAD

**Discussion by yxxhero** [guangzhengli/kubenates_workshop#10] [General]
我是helm的中国分支维护者，很高兴能看到helm相关的部分，关于helm的问题，可以咨询我，一起进一步推广helm的使用，另外我同时维护helmfile项目，用来声明式管理helm的chart，功能强大，github.com/helmfile/helmfile

**Comment by guangzhengli** [comment]
谢谢，正好有一个问题想问，helm charts 目前有没有类似于 DockerHub 公共仓库可以上传和下载使用呢，感觉官网提到的几个包括 Google Cloud Storage，JFrog，GitHub Pages 都不太适合初学者练习使用。

**Comment by yxxhero** [comment]
https://artifacthub.io/

**Reply by guangzhengli** [reply]
> https://artifacthub.io/

我理解 https://artifacthub.io/ 只提供作为发布的托管，并不提供真正存储的地方。

我看官网提供了 [GitHub Action -> GitHub Pages 来生成和存储 charts 的教程](https://helm.sh/docs/howto/chart_releaser_action/)，我可能优先用这个作为该仓库的教程，然后在 artifacthub 上面发布。

**Reply by yxxhero** [reply]
对的

**Comment by EvansYe2** [comment]
可以考虑增加 Harbor章节的结合使用 ，开源的 Docker 镜像仓库
